molecule/default/verify.yml

```diff
@@ -7,6 +7,10 @@
       register: "aptconfig"
       changed_when: false
 
+    - name: "Print apt-config variables"
+      debug:
+        msg: "{{ aptconfig.stdout }}"
```

Comment:
What is this useful for?

Reply:
This was part of the debug commit a8fbc201a15d0002283853c955f24592eff28266 which I'll revert.